backend/config/corpora-api.yml

```diff
@@ -1027,7 +1027,9 @@ paths:
                   organism_terms:
                     $ref: "#/components/schemas/wmg_ontology_term_id_label_list"
                   tissue_terms:
-                    $ref: "#/components/schemas/wmg_ontology_term_id_label_list"
+                    # TODO: specify further: https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1967
```

Comment:
Can we link to github within code and not zenhub?

On the FE we just append `#<issue-number>`
https://github.com/chanzuckerberg/single-cell-data-portal/blob/758483f5dca54b241ad5c332323cf024d52583cf/frontend/src/components/CreateCollectionModal/components/Content/components/LinkInput/index.tsx#L146

Reply:
Discussed over slack-- Issue of handling todos in code added to agenda of upcoming staff meeting